src/InputPicker/InputPicker.tsx

```diff
@@ -313,10 +314,11 @@ const InputPicker: PickerComponent<InputPickerProps> = React.forwardRef(
 
         if (creatable && item.create) {
           delete item.create;
+          onCreate?.(value, item, event);
```

Comment:
You should add corresponding unit tests for the `onCreate` prop.

Reply:
https://github.com/rsuite/rsuite#test-driven-developmenttdd

Reply:
On it

Reply:
https://github.com/rsuite/rsuite/blob/next/src/InputPicker/test/InputPickerSpec.js

Reply:
@simonguo Added tests. Having problems with `npm install` while testing. Should I submit a bug?

Reply:
There is an error in the tests, let me check.

Reply:
In addition, the documentation update instructions for onCreate are missing.